### ISSUE TITLE
Mailchimp <> email capture form are no longer integrated. Replace Mailchimp with Autopilot

### ISSUE THREAD

**Issue by willyogo** [Giveth/giveth-next#30]
As Giveth admins, we want to use one tool for all email communications rather than two. We also want to get the most out of our Autopilot integration and capture more user analytics.

Mailchimp is currently not capturing form submissions on next.giveth.io, which makes this a great opportunity to replace Mailchimp with Autopilot. Once the integration is done, I can export all of our contacts from Mailchimp and import them into Autopilot.

Autopilot makes this really easy; we just need to paste a code snippet into the <head> tag of every page. Code snippet: https://discord.com/channels/679428761438912522/769909828407001148/821180784751935519

Autopilot tracking code documentation: https://support.autopilothq.com/hc/en-us/articles/205777916-Track-user-activity-in-your-app

AC:
1. Code snippet above is added to the <head> tag of every page on giveth.io
2. The newsletter form on the homepage is auto-detected by Autopilot, and when a user enters their email on the homepage, they are added to the Newsletter list in Autopilot (Willy can help with this)

**Comment by laurenluz** [comment]
I think it would be nice to have the lil' subscribe guy appear on every page unless the user "subscribes" or clicks the "x". then he should go away

**Comment by mateodaza** [comment]
I'll configure that.

If anyone finds that annoying (like me) let me know which routes is most needed and I'll change it back

**Comment by laurenluz** [comment]
haha ok maybe just homepage and projects page?
